OEPS/OEP-5.mediawiki

```diff
@@ -103,30 +106,60 @@ If any of the transfers fail, all of the transfers must be failed, and the metho
 def approve(to, tokenId)
 </pre>
 
-
-The approve function allows the <code>to</code> address to transfer the <code>tokenId</code> NFT.
+The approve method allows the <code>to</code> address to transfer the <code>tokenId</code> NFT.
 If this function is called again it overwrites the <code>to</code> address that can transfer the NFT.
 <code>tokenId</code> should be string or bytearray.
 The <code>to</code> address must be 20-byte address.
 If not, this method must <code>throw</code> an exception.
 
-====takeOwnership====
+====getApproved====
 
 <pre>
-def takeOwnership(to, tokenId)
```

Comment:
There is no version control of OEP-5, author shall consider downward compatibility 
- do not change existing functions
- do not remove existing functions

Because this OEP is not in Final state, new features can be applied by then
- can add new functions

In this case, please consider the following options,
1. keep the original function form (name and parameters)
2. create a new OEP for non-fungible token enhancement

Reply:
I re-added the takeOwnership method to optional methods. This will resolve backwards compatibility issues and will prevent the need for an additional OEP.